.travis.yml

```
@@ -27,13 +27,15 @@ install:
   - pip install --upgrade pip
   - pip install -r requirements/requirements.txt
   - pip install -r requirements/test_requirements.txt
+  - pip install -r requirements/integration_test_requirements.txt
```

To stop the linux logs reaching the size limit, maybe making the pip install less verbose would work?
```suggestion
  - pip install -r requirements/integration_test_requirements.txt -q
```

---

That's a good idea, thanks. Will certainly give it a try.